x-pack/plugins/security_solution/public/management/components/console/components/command_execution_output.tsx

```diff
@@ -61,15 +62,19 @@ export const CommandExecutionOutput = memo<CommandExecutionOutputProps>(
       <CommandOutputContainer>
         <div>
           <UserCommandInput input={command.input} />
-          {isRunning && <EuiLoadingChart size="m" style={{ marginLeft: '0.5em' }} />}
         </div>
         <div>
+          <EuiSpacer size="s" />
+
+          {isRunning && <EuiLoadingChart size="m" style={{ marginRight: '1em' }} />}
```

Comment:
Can we use any Eui defaults for `marginRight`?

Reply:
I can look through the variables and see if there is something there